gui/qt/transaction_dialog.py

```diff
@@ -409,7 +410,11 @@ def add_io(self, vbox):
         hbox = QHBoxLayout()
         hbox.setContentsMargins(0,0,0,0)
 
-        hbox.addWidget(QLabel(_("Inputs") + ' (%d)'%len(self.tx.inputs())))
+        num_inputs = len(self.tx.inputs())
+        inputs_lbl_text = ngettext("Input", "Inputs ({num_inputs})", num_inputs)
+        if num_inputs > 1:
+            inputs_lbl_text = inputs_lbl_text.format(num_inputs=num_inputs)
```

Comment:
nit: if num_inputs is == 0, then this will render as `Inputs ({num_inputs})`.

I believe you can just call .format regardless. In the singular case, the extra keyword args to .format will be ignored. Example:
```
In [7]: '{bond}'.format(james="00", bond="7")
Out[7]: '7'
```

Reply:
FYI to anyone reading this. In our language, we say "0 Inputs'.  :)

Reply:
Oh, that is interesting. Haven't thought of that and will keep it in mind.